massgen/cli.py

```
@@ -2583,6 +2626,61 @@ def create_dspy_paraphraser_from_config(
+        or None if *base* is None.
+    """
+    if base is None:
+        return None
+    from pathlib import Path as _Path
+
+    from .logger_config import get_log_session_root as _get_root
+
+    try:
+        session_root_name = _get_root().name
+        return str(_Path(base) / session_root_name)
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)